sso-samples/oidc-sso-sample/pickup-dispatch/src/main/webapp/bcIFrame.jsp

```diff
@@ -0,0 +1,29 @@
+<html>
+<head>
+    <title>Back channel Logout Session Management RP IFrame</title>
```

Comment:
`Backchannel` should be one word. Please update the title.

Reply:
Addressed with 5a0afb6d740f15ba4e8473e191aea05c4c08dc23